### ISSUE TITLE
paddleocr 2.5.0.3 crossentropyloss print extra shape info

### ISSUE THREAD

**Issue by zhangluustb** [PaddlePaddle/PaddleOCR#6958]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Ubuntu 20.04
- 版本号/Version：Paddle：  paddleocr 2.5.0.3
- 运行指令/Command Code：
`
import paddle
paddle.seed(99999)
N=100
C=200
reduction='mean'
input =  paddle.rand([N, C], dtype='float64')
label =  paddle.randint(0, C, shape=[N], dtype='int64')
weight = paddle.rand([C], dtype='float64')
dy_ret = cross_entropy_loss(input,label)

[out]
[100, 200]
/home/zhanglu32/miniconda3/lib/python3.9/site-packages/paddle/fluid/dygraph/math_op_patch.py:251: UserWarning: The dtype of left and right variables are not the same, left dtype is paddle.float64, but right dtype is paddle.bool, the right dtype will convert to paddle.float64
  warnings.warn(
`

**Comment by MissPenguin** [comment]
It is suggested to commit a issue in PaddlePaddle official GitHub website: https://github.com/PaddlePaddle/Paddle/issues
